components/02-elements/side-menu/side-menu.hbs

```diff
@@ -1,10 +1,3 @@
 <div class="side-menu {{ class }}">
-    {{> '@list' list }}
-
-    <a href="#" class="side-menu__link link">
-        Example
-    </a>
-    <a href="#" class="side-menu__link link">
-        Sign in
-    </a>
+    {{ render '@dropdown-list--with-nested' }}
```

Comment:
shouldn't it be a module if we use another element inside this one?

Reply:
moved.